.gitignore

```diff
@@ -20,6 +20,7 @@
 # produced vignettes
 vignettes/*.html
 vignettes/*.pdf
+*.html
```

Comment:
Why was this needed? Shouldn't be any html generated except by the vignette

Reply:
The vignette does not produce a html file when I knit the file. I don't know why. So when I want to send around the html file to others I had to make a rmd file then knit to get the html. Now the checks were complaining that I had a html file somewhere, so I added this to the gitignore. But I can remove if you want